lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1499,6 +1513,32 @@ data ErrNoSuchEpoch = ErrNoSuchEpoch
     , errCurrentEpoch :: W.EpochNo
     } deriving (Eq, Show)
 
+getNetworkClock :: Handler ApiNetworkClock
+getNetworkClock = do
+    ntpS <- liftIO $ withNtpClient nullTracer ntpSettings $ \client -> do
```

Comment:
It would be helpful for users if `NtpTrace` messages were logged -- particularly because the NTP server list has been hard-coded.

Reply:
used `NtpTrace` and now when running test we have :

Reply:
this was unsatisfactory -> added `ntpClientTracer` in https://github.com/input-output-hk/cardano-wallet/pull/1396/commits/14482aba28a1b04776c03b675bbb68be3e3a824d